charts/px-backup/templates/pxcentral-pod-list.yaml

```diff
@@ -0,0 +1,27 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pods-list
+  namespace: {{ .Values.namespace }} 
```

Comment:
`ClusterRole` is not a namespace scoped object. Rename this namespace line.

Reply:
Removed.